nbconvert/templates/html/full.tpl

```diff
@@ -12,7 +12,29 @@
 <title>{{nb_title}}</title>
 
 {%- if "widgets" in nb.metadata -%}
-<script src="https://unpkg.com/jupyter-js-widgets@2.0.*/dist/embed.js"></script>
+<script>
+  (function() {
+    function addWidgetsRenderer() {
+      var mimeElement = document.querySelector('script[type="application/vnd.jupyter.widget-view+json"]');
+      var scriptElement = document.createElement('script');
+      var widgetRendererSrc = 'https://unpkg.com/@jupyter-widgets/html-manager@*/dist/embed-amd.js';
```

Comment:
Should this always be unpinned, or should we be picking a version?

Reply:
This is identical to nbviewer and it's been working well there: https://github.com/jupyter/nbviewer/blob/67ee47e55fa697614e03aa060a86019f75f2ddca/nbviewer/templates/notebook.html#L90
and that code was taken from jason's documentation:
http://ipywidgets.readthedocs.io/en/stable/migration_guides.html?highlight=migrate#updating-embedded-widgets

Reply:
Can you add some comments to explain the reasoning behind it this entire function? Also please include a pointer to jason's documentation in the comments?